extensions/rest/src/main/java/org/radargun/stages/RESTOperationsTestStage.java

```diff
@@ -25,10 +26,12 @@
  *
  * @author Martin Gencur
  */
-@Namespace(TestStage.NAMESPACE)
+@Namespace(RESTOperationsTestStage.NAMESPACE)
 @Stage(doc = "Test using RESTOperations with specific URL")
 public class RESTOperationsTestStage extends TestStage {
 
+   public static final String NAMESPACE = "urn:radargun:stages:rest:" + Version.SCHEMA_VERSION;
```

Comment:
Okay, makes sense. `rest:rest-operations-test` in XML duplicates the `rest`, but I don't know about more specific name for the operations.